vhdbuilder/packer/install-dependencies.sh

```diff
@@ -345,6 +345,8 @@ if grep -q "fullgpu" <<< "$FEATURE_FLAGS" && grep -q "gpudaemon" <<< "$FEATURE_F
 fi
 fi
 
+systemctlEnableAndStart cgroup-stats.timer || true
```

Comment:
I always forget how this works...please enable both the timer and the service for my sanity